### DISCUSSION TITLE
notes/2024-06-17-headphones/

### DISCUSSION THREAD

**Discussion by giscus** [asboyer/asboyer.github.io#24] [Comments]
# notes/2024-06-17-headphones/

Hi, I'm Andrew Boyer and this is my personal website, asboyer.com. Here you can find more about what I’m up to and what I find interesting. 

https://www.asboyer.com/notes/2024-06-17-headphones/

**Comment by BWang49** [comment]
Interesting—l’ll have to experiment myself.
